src/components/paymentJugde/PaymentJugde.tsx

```diff
@@ -0,0 +1,25 @@
+import CheckCircle from "@icons/check-circle.svg";
+import DeleteCircle from "@icons/delete-circle.svg";
+import React from "react";
+
+type paymentMethod = {
+  payment: string;
+  accepted: boolean;
+};
+
+export function PaymentJugde({ paymentMethods }: { paymentMethods: paymentMethod[] }) {
+  return (
+    <>
+      <ul className="text-xs">
+        {paymentMethods.map((item) => (
+          <li key={item.payment}>
+            <div className="flex justify-between">
+              <span>{item.payment}</span>
+              <span>{item.accepted ? <CheckCircle /> : <DeleteCircle />}</span>
+            </div>
+          </li>
+        ))}
+      </ul>
+    </>
```

Comment:
見逃しだと思うが、1つのタグで囲まれている場合は `react-fragment` なくてもOKです
```suggestion
    <ul className="w-fit">
      {paymentMethods.map((item) => (
        <li key={item.payment}>
          <div className="flex items-center justify-between space-x-4">
            <span>{item.payment}</span>
            <span>{item.accepted ? <CheckCircle /> : <DeleteCircle />}</span>
          </div>
        </li>
      ))}
    </ul>
```